DropboxLike.Domain/Repositories/File/FileRepository.cs

```diff
@@ -14,7 +14,7 @@ namespace DropboxLike.Domain.Repositories.File;
 
 public class FileRepository : IFileRepository
 {
-  private readonly string _bucketName;
+  private readonly string? _bucketName;
```

Comment:
This should not be nullable as it comes from the configuration that should be non-null.